src/components/shared/footer.cr

```diff
@@ -1,18 +1,29 @@
 class Shared::Footer < BaseComponent
   def render
-    div class: "container mx-auto px-12 pb-8 flex flex-row justify-between" do
-      div class: "-mx-2 mb-2" do
-        footer_link("Pricing")
-        footer_link("Support")
-        footer_link("Terms")
-      end
-      div do
-        para "© 2019 Bloomlist", class: "text-sm text-grey-dark"
+    footer class: "py-4" do
+      div class: "container mx-auto" do
+        div class: "flex flex-wrap justify-center md:justify-between items-center md:px-6" do
+          div class: "flex flex-col items-center md:flex-row" do
+            footer_link("Code of Conduct", "https://github.com/luckyframework/lucky/blob/master/CODE_OF_CONDUCT.md")
+            footer_link("Contributors", "https://github.com/luckyframework/lucky/graphs/contributors")
+          end
+          div class: "flex flex-wrap md:flex-no-wrap justify-between items-center py-4 md:py-0 md:px-6" do
+            footer_icon("https://github.com/luckyframework/lucky", "icons/github.svg", "Github")
+            footer_icon("https://twitter.com/luckyframework", "icons/twitter.svg", "Twitter")
+            footer_icon("https://gitter.im/luckyframework/Lobby", "icons/gitter.svg", "Gitter")
```

Comment:
I realized that instead of making it a macro like I suggested, you could use the `asset` macro here!
```suggestion
            footer_icon("https://gitter.im/luckyframework/Lobby", asset("icons/gitter.svg"), "Gitter")
```

Then you can do that for line 11/12 too and change the macro back to a private def. Sorry for the run around!

Reply:
all set and no worries.